nnpdf_data/nnpdf_data/new_commondata/CMS_1JET_8TEV/metadata.yaml

```diff
@@ -42,7 +42,7 @@ implemented_observables:
         description: jet rapidity
         label: $|y|$
         units: ''
-      pT:
+      pT2:
```

Comment:
Why did was this change made (I see the corresponding change was also done in kinematics.yaml)?

Reply:
I am not completely sure, but I think this was discussed with @scarlehoff at some point.
The reason is probably that tanishq started using pT2 instead of pT, hence just to uniform to that.

Reply:
I set it back to pT. I think this makes more sense as is the kin var for the process JET